assignments/lambda-classes.js

```diff
@@ -1 +1,136 @@
 // CODE here for your Lambda Classes
+
+class Person{
+    constructor (attributes){
+        this.name = attributes.name;
+        this.age = attributes.age;
+        this.location = attributes.location;
+        this.gender = attributes.gender;
+    }//ATR
+
+    //Methods
+    speak(){
+        return `Hello my anme is ${this.name}, I am from ${this.location}`
+    } 
+}//Person
+
+class Instructor extends Person{
+    constructor (Iattributes){
+        super(Iattributes);
+        this.specialty = Iattributes.specialty;
+        this.favLanguage = Iattributes.favLanguage;
```

Comment:
The convention for JS object variables is to spell them in *camelCase* (first letter is in lowercase; first letter of each subsequent word is in uppercase). Doing this visually distinguishes it from other Javascript entities like classes and constructor functions.